config/database.yml.txt

```diff
@@ -0,0 +1,30 @@
+# SQLite version 3.x
+#   gem install sqlite3
+#
+#   Ensure the SQLite 3 gem is defined in your Gemfile
+#   gem 'sqlite3'
+#
+default: &default
+  adapter: postgresql
+  pool: 5
+  timeout: 5000
+
+development:
+  adapter: postgresql
+  encoding: unicode
+  database:
+  pool: 5
+  username:
```

Comment:
I suggest you to not specify username for a development database, to simplify local deployment.